pages/search.tsx

```diff
@@ -377,7 +377,10 @@ export default function Search({
                   aria-haspopup="true"
                   aria-expanded="true"
                 >
-                  {sort ? `Sort: ${sort}` : 'Relevance'}
+                  {sort &&
+                  Object.prototype.hasOwnProperty.call(SORT, sort as string)
```

Comment:
I think it's fine to do a simpler check here

Reply:
making the changes!